src/core_plugins/elasticsearch/lib/kibana_index_mappings.js

```diff
@@ -1,16 +1,24 @@
-export const mappings = {
+const currentMappings = {
```

Comment:
👍 can do, can you explain why?  I've seen this before, maybe its worth adding to our style guide.